### PR TITLE
OI-1146 (Add ability to use module csv-parse in the IDE) - i…

### PR DESCRIPTION
**Pull request by savkaoleg**
…mplement as methods in "utils" module.

### CODE REVIEW

File: src/ox_modules/module-utils.js [hunk 1]
```diff
@@ -42,4 +44,64 @@ export default class UtilsModule extends OxygenModule {
         utils.assertArgumentNumberNonNegative(ms, 'ms');
         deasync.sleep(ms);
     }
+
+    /**
+     * @summary Reads data from csv file
+     * @function readCsv
+     * @param {String} filePath - Absolute path to file
+     * @param {Object} options - Options (https://csv.js.org/parse/options/)
```
Comment (romovs) [inline src/ox_modules/module-utils.js:52]: If this argument is optional, then it should be marked as such `=`

File: src/ox_modules/module-utils.js [hunk 2]
```diff
@@ -42,4 +44,64 @@ export default class UtilsModule extends OxygenModule {
...
+        }
+
+        return table;
+    }
+
+    /**
+     * @summary Writes data into csv file
+     * @function writeCsv
+     * @param {String} filePath - Absolute path to file
+     * @param {Array} data - Absolute path to file
```
Comment (romovs) [inline src/ox_modules/module-utils.js:95]: Is `Array` type correct? Also description is wrong.

File: src/ox_modules/module-utils.js [hunk 3]
```diff
@@ -42,4 +44,64 @@ export default class UtilsModule extends OxygenModule {
...
+     * @param {Object} options - Options (https://github.com/anton-bot/objects-to-csv#async-todiskfilename-options)
```
Comment (romovs) [inline src/ox_modules/module-utils.js:96]: If options is optional, the it should be marked as such

**Review by romovs — CHANGES_REQUESTED** [review]

File: src/ox_modules/module-utils.js (hunk 1 above)
Comment (savkaoleg) [reply src/ox_modules/module-utils.js]: fixed

File: src/ox_modules/module-utils.js (hunk 2 above)
Comment (savkaoleg) [reply src/ox_modules/module-utils.js]: fixed

File: src/ox_modules/module-utils.js (hunk 3 above)
Comment (savkaoleg) [reply src/ox_modules/module-utils.js]: fixed

**Pull request merged**